module/Finna/src/Finna/RecordDriver/SolrLido.php

```diff
@@ -195,20 +196,17 @@ public function getAllImages($language = 'fi')
             // Process rights first since we may need to duplicate them if there
             // are multiple images in the set (non-standard)
             $rights = [];
-            if (isset($resourceSet->rightsResource)) {
-                foreach ($resourceSet->rightsResource as $rightsResource) {
-                    if (!empty($rightsResource->rightsType->conceptID)) {
-                        $conceptID = $rightsResource->rightsType->conceptID;
-                        $type = strtolower((string)$conceptID->attributes()->type);
-                        if ($type == 'copyright') {
-                            $rights['copyright'] = (string)$conceptID;
-                            $link = $this->getRightsLink(
-                                strtoupper($rights['copyright']), $language
-                            );
-                            if ($link) {
-                                $rights['link'] = $link;
-                            }
-                        }
+            if (!empty($resourceSet->rightsResource->rightsType->conceptID)) {
```

Comment:
Sama kuin yllä. Ei voi lähteä siitä, että näitä elementtejä on vain yksi ja copyright löytyy siitä.